javascript-bank/bank.js

```diff
@@ -0,0 +1,37 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (balance > 0) {
+    this.accounts.push(new Account(this.nextAccountNumber, holder));
+    this.accounts[this.nextAccountNumber - 1].deposit(balance);
+    this.nextAccountNumber++;
+  } else {
+    return null;
+  }
+  return this.nextAccountNumber - 1;
+};
+
+Bank.prototype.getAccount = function (number) {
+  for (var i = 0; i < this.accounts.length; i++) {
+    if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    }
+  }
+  return null;
+};
+
+Bank.prototype.getTotalAssets = function () {
+  var total = 0;
+  if (this.accounts.length === 0) {
```

Comment:
This conditional is unnecessary. If the length of the array is zero, your loop following this will end immediately, and leave you in the same situation as your condition is preparing for :+1: